### PR TITLE
Add status to account.

### PR DESCRIPTION
**Pull request by marcushanses**

### CODE REVIEW

File: Client/Accounts/Status.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { gracely } from "gracely"
+import { http } from "cloudly-http"
+import * as rest from "cloudly-rest"
+import { Account } from "../../Account"
+
+export class Status extends rest.Collection<gracely.Error> {
```
Comment (AntonCroner) [inline Client/Accounts/Status.ts:6]: remove rest collection

**Review by AntonCroner — CHANGES_REQUESTED** [review]

File: Account/Status.ts [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { isly } from "isly"
+
+export type Status = {
+	mode: "active" | "frozen" | "closed"
```
Comment (SiFalorSum) [inline Account/Status.ts:4]: Use the `Mode` and a `Reason` type. Reason can be defined as 
```
export type Reason = "overdraft" | "other" | (string & Record<never, never>)
```
to fool typescript into giving type suggestions

**Review by SiFalorSum — APPROVED** [review]

File: Client/Accounts/Status.ts (hunk 1 above)
Comment (marcushanses) [reply Client/Accounts/Status.ts]: Done.

File: Account/Status.ts (hunk 2 above)
Comment (marcushanses) [reply Account/Status.ts]: Check!

**Review by AntonCroner — APPROVED** [review]

**Pull request merged**